R/process.R

```diff
@@ -548,12 +552,12 @@ process_create_context_ids <- function(data, contexts) {
     for (v in vars) {
       id_link[[v]] <-
         xxx %>%
-        dplyr::rename(dplyr::all_of(c(find = v))) %>%
-        dplyr::select(dplyr::all_of(c("find", "id"))) %>%
+        dplyr::rename(dplyr::all_of(c(value = v))) %>%
```

Comment:
I can't recall enough of the logic here. Can you elaborate on why we should be suing value instead of find?

Reply:
On [Line 501](https://github.com/traitecoevo/traits.build/blob/eb4a62ed0efe066744e71af95eb2f4fa172c88ef/R/process.R#L501), I filtered for rows where the `find` column is not NA. If there are rows where the `find` column is not NA, I create a named vector with the `value` column as the values and `find` column as the names. Then on [Line 506](https://github.com/traitecoevo/traits.build/blob/eb4a62ed0efe066744e71af95eb2f4fa172c88ef/R/process.R#L506), it uses this named vector to replace the `find` context values with `value` values. We are now working completely with `value` values and can ignore the `find` column. 

That's why on [Line 554](https://github.com/traitecoevo/traits.build/blob/eb4a62ed0efe066744e71af95eb2f4fa172c88ef/R/process.R#L554) it's now creating link values via the `value` column. We don't want to create link values via the `find` column because the `find` column doesn't contain the final unique context values. 

Let me know if it doesn't make sense!